### PR TITLE
Export everything under named entrypoint

### PR DESCRIPTION
**Pull request by razor-x**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,14 +7,45 @@ TypeScript types for the Seam API.
...
+The main entrypoint for this module is empty: there is dedicated entrypoint for each Seam API namespace.
+
+Each directory under `src/lib/seam/*` is owned by an upstream Seam repository.
+These files should not be edited manually as they will be overridden by automation.
+
+Each entrypoint may export one or more of the following:
+
+- Zod schemas: collectively exported as a single object named `schemas`.
+- Types: derived directly from the [Zod Schemas][zod] and exported at the top level.
+- A separate Zod schmea named `routes` generated by [nextlove] and exported at the top level.
```
Comment (razor-x) [inline README.md:27]: @codetheweb Is this what you are thinking?

File: README.md [hunk 2]
```diff
@@ -7,14 +7,45 @@ TypeScript types for the Seam API.
...
+- A type named `Routes` generated by [nextlove].
```
Comment (razor-x) [inline README.md:28]: I was going to link to the nextlove docs on the structure of this but I think we only document the openapo spec?

File: README.md [hunk 3]
```diff
@@ -7,14 +7,45 @@ TypeScript types for the Seam API.
...
+have a single source of truth that stays up to date with the API.
+
+### Structure
+
+The main entrypoint for this module is empty: there is dedicated entrypoint for each Seam API namespace.
+
+Each directory under `src/lib/seam/*` is owned by an upstream Seam repository.
+These files should not be edited manually as they will be overridden by automation.
+
+Each entrypoint may export one or more of the following:
```
Comment (razor-x) [inline README.md:23]: Should we export the openapi json (as a plain JS object)? @codetheweb

Comment (razor-x) [reply README.md:23]: Maybe better to point users at the hosted file? We could export `openapiUrl` 🤔

Comment (codetheweb) [reply README.md:23]: yeah I think we should export it

personally I like having everything versioned under the same place

File: README.md (hunk 2 above)
Comment (codetheweb) [reply README.md:28]: [typed-axios](https://github.com/seamapi/typed-axios) documents the structure of the route type

**Comment by razor-x** [comment]
@codetheweb Updated the allowed export docs, look ok now?

**Review by codetheweb — APPROVED** [review]
lgtm

**Pull request merged**